packages/odyssey-react/.storybook/main.js

```diff
@@ -47,8 +47,11 @@ module.exports = {
                 @import '@okta/odyssey/src/scss/base/reset';
                 @import '@okta/odyssey/src/scss/base/accessibility';
                 @import '@okta/odyssey/src/scss/base/iconography';
-                @import '@okta/odyssey/src/scss/base/typography';
-
+                @import '@okta/odyssey/src/scss/base/typography-global';
+                @import '@okta/odyssey/src/scss/base/typography-text';
+                @import '@okta/odyssey/src/scss/base/typography-list';
+                @import '@okta/odyssey/src/scss/base/typography-header';
```

Comment:
These were the imports previously found in `@okta/odyssey/src/scss/base/typography`, it also included a `typography-list` partial which is no longer necessary as this PR includes the module for it.

Reply:
Thanks for the comment - eventually we'll have to work toward removing everything here but we are making slow and steady progress!